src/terminal/terminal/terminal.h

```diff
@@ -21,11 +21,16 @@
 #ifndef _GUAC_TERMINAL_H
 #define _GUAC_TERMINAL_H
 
+/**
+ * Constants, structures, and function definitions defining the core
+ * functionality for the terminal library.
+ *
+ * @file terminal.h
+ */
+
 #include "config.h"
 
 #include "buffer.h"
```

Comment:
Are any of these remaining private headers still needed here (`buffer.h`, `display.h`, `scrollbar.h`, etc.)? If internal, they shouldn't be part of the public header, and the headers themselves should not be installed (they should be listed as "noinst" in `Makefile.am`).

Reply:
Nope! They shouldn't be! Fixed.